src/rest-api/endpoints/management-endpoint/elasticsearch-endpoint.md

```diff
@@ -11,6 +11,10 @@ toc: true
 tree_item_index: 400
 ---
 
+{{#> callout type='warning' heading='Deprecated Endpoint in LTS 2025'}}
+Since LTS 2025 supports multiple Search Engines, this endpoint is no longer relevant and is replaced by the new [Search management endpoint]({{page page='search-endpoint'}}).
```

Comment:
```suggestion
Since LTS 2025, Nuxeo supports multiple Search Engines, this endpoint is no longer relevant and is replaced by the new [Search management endpoint]({{page page='search-endpoint'}}).
```